src/main/java/dev/andrylat/vomelianchuk/finances/mortgagecalculator/calculation/MortgageCalculator.java

```diff
@@ -0,0 +1,38 @@
+package dev.andrylat.vomelianchuk.finances.mortgagecalculator.calculation;
+
+import dev.andrylat.vomelianchuk.finances.mortgagecalculator.exceptions.DataMortgageCalculatorException;
+
+import java.util.List;
+
+public class MortgageCalculator {
+    private static final String INCORRECT_NUMBERS =  "The numbers are not correct";
+    private final static double paymentsPerYear = 12;
+    private final double totalPrice;
+    private final double downPayment;
+    private double interestRate;
+    private final double numberOfYears;
+
+    public MortgageCalculator(List<Double> dataMortgage) {
+        totalPrice = dataMortgage.get(0);
```

Comment:
NOPE. Never do it in this way! It is not clear why you should have total price on the first place,  payment on second and so on. Always create data object for transfering data. DTO

Reply:
Fixed here: d7b1b80